Components documentation improvements (#2121)
withastro/starlight: docs/src/content/docs/components/cards.mdx

---
title: Cards

> description: Learn how to display content in a box with the Card component.

Reviewer: Maybe we could rewrite the description to more closely resemble the others?
```suggestion
description: Learn how to cards in Starlight to display content in a box.
```
